From 4cb88d2947853896d109e3955d25f90efaa10571 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Mon, 27 Nov 2017 19:01:23 +0000
Subject: [PATCH 1/1] Add 'StartupWMClass' to desktop file

* Fixes pinning to panel in budige and other system integration thingies.

Signed-off-by: Joey Riches <josephriches@gmail.com>
---
 data/org.kde.kdenlive.desktop | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/org.kde.kdenlive.desktop b/data/org.kde.kdenlive.desktop
index fec333d..c1859bb 100644
--- a/data/org.kde.kdenlive.desktop
+++ b/data/org.kde.kdenlive.desktop
@@ -101,6 +101,7 @@ X-DocPath[cs]=http://cs.wikibooks.org/wiki/Kdenlive
 Terminal=false
 MimeType=application/x-kdenlive;
 Categories=Qt;KDE;AudioVideo;AudioVideoEditing;
+StartupWMClass=Kdenlive
 Keywords=editing;video;audio;mlt;kde;
 Keywords[ca]=edició;vídeo;àudio;mlt;kde;
 Keywords[ca@valencia]=edició;vídeo;àudio;mlt;kde;
-- 
2.15.0

